src/labkey/Query.ts

```diff
@@ -25,6 +25,9 @@ import { deleteRows, insertRows, saveRows, selectDistinctRows, selectRows, trunc
 import * as SQL from './query/experimental/SQL'
 import * as Visualization from './query/Visualization'
 import { Filter } from './filter/Filter'
+import {request} from "./Ajax";
```

Comment:
I'd like to discuss moving the implementation out of `labkey/Query.ts` and into a subfile of `src/labkey/query`. It allows for `labkey/Query.ts` to act more as a exportable namespace free of implementation.